automation_tools/__init__.py

```diff
@@ -418,6 +418,11 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_scap_client():
+    """Task to setup puppet-foreman_scap_client."""
+    run('yum -y puppet-foreman_scap_client', warn_only=True)
```

Comment:
shouldn't it be "yum -y install puppet-foreman_scap_client" , I think we are missing "install", is it so ?